main.py

```diff
@@ -516,15 +522,11 @@ def load_evcs(self, event):
                     log.info(
                         f'Could not load EVC {circuit_id} because {exception}')
                     continue
-                log.info(f'Loading EVC {circuit_id}')
-                if evc.archived:
-                    continue
-                new_evc = self.circuits.setdefault(circuit_id, evc)
-                if new_evc == evc:
-                    if evc.is_enabled():
-                        log.info(f'Trying to deploy EVC {circuit_id}')
-                        evc.deploy()
-                    self.sched.add(evc)
+                evc.deactivate()
+                evc.current_path = Path([])
+                evc.sync()
+                self.circuits.setdefault(circuit_id, evc)
+                self.sched.add(evc)
```

Comment:
@ajoaoff, please add a comment here explaining why we need to deactivate+sync when loading EVCs from the Storehouse.

Reply:
When the EVCs are loaded from the storehouse, they are no active, but the state in the storehouse may be wrong depending on how Kytos was terminated. So, the EVC is disabled, and then synced to the storehouse.
After that, a routine will try to deploy all the enabled EVCs.

Reply:
Sorry, I meant that you should add a comment _in the code_ explaining why this behavior is necessary.